Example/YelpAPI.xcodeproj/project.pbxproj

```diff
@@ -124,6 +128,8 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				0B5080851CBEEA950071508E /* OHHTTPStubs.framework in Frameworks */,
+				0B5080831CBEEA800071508E /* OCMock.framework in Frameworks */,
```

Comment:
If you're up for it, you could nuke the YelpAPI_Tests target here since there's nothing in it and it's just causing confusion.